src/coreclr/vm/gcheaputilities.cpp

```diff
@@ -334,8 +335,7 @@ HRESULT GCHeapUtilities::LoadAndInitialize()
     assert(g_gc_load_status == GC_LOAD_STATUS_BEFORE_START);
     g_gc_load_status = GC_LOAD_STATUS_START;
 
-    LPWSTR standaloneGcLocation = nullptr;
-    CLRConfig::GetConfigValue(CLRConfig::EXTERNAL_GCName, &standaloneGcLocation);
+    LPCWSTR standaloneGcLocation = Configuration::GetKnobStringValue(W("System.GC.GCName"), CLRConfig::EXTERNAL_GCName);
```

Comment:
was debating between `System.GC.Name` and `System.GC.GCName`, but thought it should be consistent with `DOTNET_GCName`?

Reply:
generally we drop the `GC` part in the runtime config since it already starts with `System.GC`.

Reply:
ok will change after the CI finishes.

Reply:
there is one CI failure in the `monointerpreter` macOS leg which is due to this change. Dont believe the change should be affecting the scenario, but the test infra is most likely failing due to it (perhaps they are using GCName or something.

Reply:
updated the name to `System.GC.Name`